portal/v2/src/ClusterDetailListComponents/Statistics/StatisticsComponent.tsx

```diff
@@ -0,0 +1,165 @@
+import { useEffect, useState } from "react"
+import { Stack, StackItem, IStackProps} from '@fluentui/react';
+import { Spinner, SpinnerSize } from '@fluentui/react/lib/Spinner';
+import { ILineChartPoints, LineChart, ILineChartDataPoint } from '@fluentui/react-charting';
+import { IChartProps } from '@fluentui/react-charting';
+import { DefaultPalette} from '@fluentui/react/lib/Styling';
+import { IMetrics} from './StatisticsWrapper';
+import { convertToUTC } from "./GraphOptionsComponent";
+
+export function StatisticsComponent(props: {
+  metrics: IMetrics[],
+  clusterName: any,
+  duration: string,
+  height: number,
+  width: number,
+  endDate: Date,
+}) {
+  const width = props.width
+  const height = props.height
+  const [points, setPoints] = useState<ILineChartPoints[]>([])
+  const [data, setData] = useState<IChartProps>({})
+  const [spinnerVisible, setSpinnerVisible] = useState<boolean>(true)
+  const timeFormat = '%H:%M'
+
+  const colors: string[] = [
+    DefaultPalette.blue,
+    DefaultPalette.blueLight,
+    DefaultPalette.blueDark,
+    DefaultPalette.blueMid,
+    DefaultPalette.black,
+    DefaultPalette.red,
+    DefaultPalette.redDark,
+    DefaultPalette.yellow,
+    DefaultPalette.yellowDark,
+    DefaultPalette.yellowLight,
+    DefaultPalette.green,
+    DefaultPalette.greenLight,
+    DefaultPalette.greenDark,
+    DefaultPalette.purple,
+    DefaultPalette.purpleLight,
+    DefaultPalette.purpleDark,
+    DefaultPalette.orange,
+    DefaultPalette.orangeLight,
+    DefaultPalette.orangeLighter,
+    DefaultPalette.magenta,
+    DefaultPalette.magentaDark,
+    DefaultPalette.magentaLight,
+    DefaultPalette.themePrimary,
+    DefaultPalette.neutralPrimary,
+    DefaultPalette.neutralDark,
+    DefaultPalette.neutralSecondary,
+    DefaultPalette.neutralTertiary,
+    DefaultPalette.teal,
+    DefaultPalette.tealDark,
+    DefaultPalette.tealLight,
+    DefaultPalette.accent,
+    DefaultPalette.themeDarker,
+    DefaultPalette.themeDarkAlt,
+    DefaultPalette.themeDark,
+    DefaultPalette.themeLight,
+    DefaultPalette.themeLighter,
+    DefaultPalette.themeLighterAlt,
+    DefaultPalette.themePrimary,
+    DefaultPalette.themeSecondary,
+    DefaultPalette.themeTertiary
+  ]
+
+  const _onLegendClickHandler = (selectedLegend: string | null | string[]): void => {
+    if (selectedLegend !== null) {
+      console.log(`Selected legend - ${selectedLegend}`);
```

Comment:
We shouldn't be committing console.logs

Reply:
Okay, yaa anyways we are not doing anything right now at a Legend click so I removed the whole function itself.